File: bots/echo_bot.go
package bots

import (
	"github.com/kraevskii-m/MailBot/botLib"
	"log"
	"time"
)

var address = "echobot-mailbot@yandex.ru"

func EchoBotController(token string) {
	bot := botLib.NewMailBot(token, "EchoBot")
	for {
		time.Sleep(5 * time.Second)
		messages, err := bot.GetUpdates(0, 0) //todo use offset and limit
		if err != nil {
			log.Print(err)
		}
		for _, message := range messages {
			botLib.NewMessage(address, message.From, message.Subject, message.Body)
		}
	}
}

File: bots/echo_bot.py
import botLibPy.bot_lib as bot_lib

TOKEN = ""
address = "echobot-mailbot@yandex.ru"

bot = bot_lib.Bot(TOKEN)
while True:
    messages = bot.get_updates()
    for message in messages:
        bot.send_message(bot_lib.Message(address, message.fr, message.subj, message.body))
